source/js/donate-modal/donate-modal.scss

```diff
@@ -1,5 +1,5 @@
 .donate-modal {
-  $modal-z-index: 10;
+  $modal-z-index: 1040; //War of escalation with .sticky-top
```

Comment:
Yeah let's not do this. We're in sass, let's use a calc like `calc(#{$sticky-index} + 1)` =)

Reply:
Ok fair enough. Turns out Bootstrap has a thing built in for this already. See updated PR.